crates/context/src/lib.rs

```diff
@@ -294,6 +298,7 @@ impl ContextManager {
     pub async fn join_context(
         &self,
         identity_secret: PrivateKey,
+        sender_secret: PrivateKey,
```

Comment:
same

Reply:
this was the third comment in the review, somehow it got broken up, crap internet 😪